_works/2025-maybauer-finn/02-titles-and-research-questions.md

```diff
@@ -0,0 +1,35 @@
+# Titel und Forschungsfrage
+
+Bitte notieren Sie mindestens 10 mögliche Titel der Arbeit und mindestens 5 mögliche Forschungsfragen/Zielsetzungen in dieses Dokument.
+
+## Titel
+
+### Iteration 0
+
+> [!NOTE]  
+> Titel generiert über ChatGPT 4o. Prompt: "Generiere 10 mögliche Titel für eine Masterarbeit in der Medieninformatik mit folgendem Expose: <[Inhalt des Exposes](03-expose.md)"
+
+* Barrierefreiheit in Unternehmen: Ein Reifegrad-Ansatz zur Umsetzung der EU-Norm EN 301 549
+* Von Compliance zu Inklusion: Strategien zur Messung und Steigerung der Barrierefreiheitsreife
+* Reifegradmodelle für barrierefreie digitale Produkte: Herausforderungen und Chancen
+* Barrierefreiheit als Prozess: Entwicklung eines Werkzeuges zur Maturity-Messung
+* Inklusive Digitalisierung: Methoden zur Steigerung der Barrierefreiheit in Organisationen
+* Workshop oder Web-Tool? Ansätze zur Prozessoptimierung in der Barrierefreiheit
+* Maturity-Modell-Ansätze für die Barrierefreiheit: Eine Fallstudie
+* Barrierefreiheits-Compliance: Werkzeuge und Methoden zur organisatorischen Vorbereitung
+* Messbare Inklusion: Ein Reifegradmodell zur Umsetzung der EU-Norm EN 301 549
+* Digitale Barrierefreiheit im Fokus: Entwicklung einer Methodik zur Prozessverbesserung
+
+### Iteration 1
+
+* Barrierefreiheit in Unternehmen: Messung und Steigerung der Reife durch Maturity-Modelle
+* Implementierung der EU-Norm EN 301 549: Reifegradmodelle als Werkzeug zur Stärkung der Barrierefreiheit in Organisationen
```

Comment:
Vielleicht besser der Name, statt die Norm ID … oder beides?

Reply:
Ja. Obwohl es in der "Szene" fast schon ein Buzz-Wort ist, so wie ISO 9241 in der MCI. Barrierefreiheits-Stärkungsgesetz (BFSG) wäre aber wahrscheinlich sprechender.